src/services/edusharing/logic/connector.js

```diff
@@ -0,0 +1,165 @@
+const request = require('request-promise-native');
+const REQUEST_TIMEOUT = 8000; // ms
+
+// STACKOVERFLOW BEAUTY
+function validURL(str) {
+	const pattern = new RegExp(
+		'^(https?:\\/\\/)?' +
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['+' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)